script/dump_db

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+cd $DEPLOYMENT_DIR
+sudo -u postgres pg_dump -U postgres -d $PG_SIMPLECMS_PROD >"dump/$(date).dump"
```

Comment:
The reason I prefer S3 is that it has (virtually) infinite storage, that is, there is zero overhead as more dumps are created. On the other hand, if the dumps are stored on EC2, then we either need to manually delete those dumps or resize the storage (which takes quite a bit of time).